### ISSUE TITLE
fullTemplateTypeCheck support

### ISSUE THREAD

**Issue by FDIM** [angular/vscode-ng-language-service#443]
I may have ran into an issue or a unsupported configuration

Is fullTemplateTypeCheck angular compiler option supposed to work?

UI: 
![image](https://user-images.githubusercontent.com/1985429/69228547-0c68ba00-0b84-11ea-954c-fa26cec3cced.png)

During AOT build I am getting following error:
```
ERROR in .../src/lib/calendar/calendar.component.html(104,9): Expected 0 arguments, but got 1.
```
Basically `removeHighlightDate` does not accept any params and typescript compilation fails, but language service does not indicate this.

Thanks for a great extension overall!

**Comment by ayazhafiz** [comment]
I think this is because we don't check the call signature of the component methods used in a template.

**Comment by kyliau** [comment]
Full template type checking is a feature planned for version 10. Please stay tuned!

**Comment by congwenma** [comment]
https://marketplace.visualstudio.com/items?itemName=Angular.ng-template

**Comment by msklvsk** [comment]
I’m getting no indication for `fullTemplateTypeCheck` errors in Angular 10. Is this still a todo?

**Comment by lordgreg** [comment]
hi. this is still not working for me. is there any additional configuration needed?

**Comment by kyliau** [comment]
Sorry, this feature is not yet available, we initially planned it for version 10, but it got delayed. We are close to finishing the project though. I'll update this thread once we are ready for an announcement! 

To clarify, this is not a feature on its own, rather an entire overhaul of the language service backend to switch from View Engine to the Ivy-native compiler.

**Comment by kyliau** [comment]
This has been fixed by the new Ivy-native language service, released in [v11.1.0](https://github.com/angular/vscode-ng-language-service/releases/tag/v11.1.0). It's an opt-in for now, please give it a try and let us know if you have any feedback.
